pkg/admin/prerun/backups.go

```diff
@@ -62,13 +62,18 @@ func (bs Backups) has(backup data.BackupName) bool {
 }
 
 func (bs Backups) removeAll(dataManager data.Manager) {
-	klog.Info("Preparing to prune backups")
+	if len(bs) == 0 {
+		return
```

Comment:
Should we add a message here that there are no backups to prune?

Reply:
Wouldn't we benefit from a message before "return"? I mean, there's Preparing and then nothing at the moment.